app/iht/viewmodels/application/home/IhtHomeRowViewModel.scala

```diff
@@ -49,13 +49,25 @@ object IhtHomeRowViewModel {
     new IhtHomeRowViewModel(deceasedName = s"${ihtApp.firstName} ${ihtApp.lastName}",
       ihtRefNo = ihtApp.ihtRefNo,
       dateOfDeath = ihtApp.dateOfDeath.toString(IhtProperties.dateFormatForDisplay),
-      currentStatus = CommonHelper.formatStatus(currentStatus),
+      currentStatus = getApplicationStatusMessage(currentStatus),
       linkLabel = getLinkLabel(currentStatus),
       link = getLink(currentStatus, ihtRef),
       linkScreenreader = getLinkScreenreader(currentStatus, s"${ihtApp.firstName} ${ihtApp.lastName}")
     )
   }
 
+  private def getApplicationStatusMessage(currentStatus: String) = {
```

Comment:
Discussed. This function has been added to ensure that application status is pulled from messages file instead of constants, to allow for Welsh translation.